nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/ui/canvas/items/flow/change-version-dialog/change-version-dialog.html

```diff
@@ -0,0 +1,89 @@
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~    http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<h2 mat-dialog-title>Change Version</h2>
+<div class="change-version" tabindex="0">
+    <mat-dialog-content>
+        <div class="flex flex-col gap-y-4">
+            <div>
+                <div>Registry</div>
+                <div class="value">{{ versionControlInformation.registryName }}</div>
```

Comment:
Since you merged my semantic color PR you need to update this to be:
```suggestion
                <div class="mat-accent font-medium">{{ versionControlInformation.registryName }}</div>
```

Reply:
There is a lot of white space on the right hand side of this dialog... maybe we could move the flow name and current version up to balance it out?

Reply:
Good call, how's this?
<img width="772" alt="Screenshot 2024-04-10 at 9 02 13 AM" src="https://github.com/apache/nifi/assets/713866/6eb46d1a-c7f5-427e-a9ac-0c8db2872b1a">